docs/smart-contracts/creating.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Creating smart contracts
+author: 'Yuxin Li'
+lastUpdated: 6th November 2023
+---
+## Introduction
+Creating smart contracts on blockchain making sure faster execution and reduced costs by eliminating the need for intermediaries. This guide provides step-by-step instructions for creating smart contracts on the Tezos blockchain. You can find additional resources on testing and deploying your smart contract in subsequent chapters.
```

Comment:
This first sentence doesn't scan for me -- not sure what it's saying. Also, don't refer to "subsequent chapters" because this is task-based doc and we don't make users read it in any specific order.